packages/codegen-ui-react/lib/react-studio-template-renderer.ts

```diff
@@ -837,6 +843,42 @@ export abstract class ReactStudioTemplateRenderer extends StudioTemplateRenderer
     );
   }
 
+  /**
```

Comment:
not sure I'm seeing the difference between this and `buildOverridesFromVariantsAndRestProp`

Reply:
`buildOverridesFromVariantsAndRestProp` is used when a 'breakpoint' variant exists on a component as it includes the default breakpoint mapping as an argument into `getOverridesFromVariants`. This function builds the call without that mapping for components that do not have a 'breakpoint' variant.

Reply:
Merged functions while conditioning on `hasBreakpoints`